src/utils/index.ts

```diff
@@ -82,7 +83,7 @@ export function getEtherscanLink(
 }
 
 export function getEtherscanLinkText(chainId: ChainId): string {
-  return NETWORKS_INFO[chainId].etherscanName
+  return t`View transaction` // NETWORKS_INFO[chainId].etherscanName
```

Comment:
remove comment

Reply:
no need to keep it

Reply:
ah, you fix wrong place

Reply:
why it's wrong, bro?

Reply:
I think he want to change all place

Reply:
![image](https://user-images.githubusercontent.com/23650671/186129493-18d2cef8-234a-4791-926d-9b2622108d5d.png)
you changed 3 place

Reply:
I dont think AddressInputPanel is correct one